Dockerfile

```diff
@@ -21,6 +21,4 @@ RUN pip install --no-cache-dir -r requirements.txt
 EXPOSE 8080
 
 ENV DEBUG_DASH False
-ENV PORT_APP 8080
-
-CMD python app.py
```

Comment:
why did you delete this command?

Reply:
sorry for that, this may be a mistake due to merging code